### PR TITLE
[POC] IDT 1 & 2 Includes

### LINKED ISSUE #5686: Prototype STY support for IDT1 & IDT2 Includes
**Is your feature request related to a problem? Please describe.**
Following the discussions had around having support for includes content in STY pages, POC's (https://github.com/bbc/simorgh/pull/5630 & https://github.com/bbc/simorgh/pull/5628) were created to demonstrate how we could support it, but those were limited to just visual journalism typed includes. Since we would also be supporting other include types (IDT1 & IDT2), we would also like to prototype rendering these include types.

**Describe the solution you'd like**
- Create a proof-of-concept branch that showcases rendering IDT1 & IDT2 typed includes on a story page. 
- Use fixture data that mimics the response payload from calling an include URL. liaise with @lizcameron on this.

**Describe alternatives you've considered**

**Testing notes**
What could go wrong? Please don't take that as a challenge.

**Additional context**

|         | IDT1          | IDT2  |
| ------------- |:-------------:| -----:|
| pal example      | https://www.bbc.co.uk/news/world-49343432 | https://www.bbc.com/turkce/haberler-turkiye-49563028 |
| content  | Quizzes   |  Datapics, Maps, Tables, Charts, Quotepics |
|  Notes  |   Interactive content, so needs to be embedded as HTML  |  Essentially images so if the PoC doesn't work for IDT2, we could look into rendering IDT2s as image blocks instead, and progressively enhance this later. |
|  block example |     `{"required" : false, "tile": "Quiz of the week 16 August 2019", "href": "/indepthtoolkit/quizzes/../.png", "platform": "highweb", "type": "include"}`| `{"required": false, "tile": "Map of Iraq showing Karbala", "href": "/idt2/adeff9e6-c40f-4913-9232-1a3a0af4af49", platform: "highweb", "type": "include"}` |

### PR DESCRIPTION
**Pull request by rhenshaw56**
Resolves 

**Overall change:** 
_Builds on the work on https://github.com/bbc/simorgh/pull/5630 to render includes for Idt1 and 2._

**Code changes:**
- _Adds Markups for IDT1 and 2._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by andrewscfc — COMMENTED** [review]
Just a question, been off for a week so coming in cold!

File: src/app/containers/Include/index.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  include: visMarkup,
+  idt2: id2Markup,
+  indepthtoolkit: idT1Markup,
+};
+
+const Include = ({ href }) => {
+  const type = href.split('/')[1];
+  return (
+    <GridItemConstrainedMedium>
+      <div dangerouslySetInnerHTML={{ __html: markupToRender[type] }} />
```
Comment (andrewscfc) [inline src/app/containers/Include/index.jsx]: Did we make a decision to accept the security risk of using `dangerouslySetInnerHTML`? Or are we intending to sandbox this in an iframe?

Comment (rhenshaw56) [reply src/app/containers/Include/index.jsx]: @andrewscfc It was agreed to build on the [shadow dom approach](https://github.com/bbc/simorgh/pull/5630) done by Rhys and Chris for VJ's just to try having similar implementations for the different include types.

Comment (andrewscfc) [reply src/app/containers/Include/index.jsx]: Thanks for that link, the key point in the description is this:

> Shadow DOM is not supported in IE11, upon a quick look, it appears that the include falls back to an iframe.

So we `dangerouslySetInnerHTML` to include some code that uses shadow dom when possible and where not possible, uses an iframe, that sounds a pretty satisfactory approach.

I mentioned security earlier, I'm not sure this is an issue actually, we will be hitting our own endpoint on the BBC domain so we will be 'dangerously inserting' html we can be confident comes from the BBC; without a cert breach, no one should be able to impersonate our endpoint.

File: src/app/lib/logger.node.js [hunk 2]
```diff
@@ -24,8 +24,8 @@ const fileTransport = new (transports.File)({
   humanReadableUnhandledException: true,
   json: true,
   level: LOG_LEVEL,
-  maxFiles: 1,
-  maxsize: 104857600, // 100MB
+  maxFiles: 10,
+  maxsize: 2, // 100MB
```
Comment (mykeels) [inline src/app/lib/logger.node.js:28]: Why is this reduced to 2?

**Comment by andrewscfc** [comment]
Questions:
- Should we implement a fallback when the IDT request responds with an error?
- If we want to support serverside rendering, we must use a class component rather than functional component using hooks as hooks only run clientside?

Assumptions:
- We will make the IDT `html` request
- We do not need to support legacy content on amp - no amp support required for IDT1

Next Steps:
- Investigate getting the requirejs fallback to function to allow IDT1 includes to display correctly.

**Comment by andrewscfc** [comment]
https://github.com/bbc/simorgh/pull/5707/commits/e4ba13ecfb6820294d9bda0efef7806d7b578ecf



This commit allows an IDT1 quiz to render by conditionally loading requirejs for assets including an IDT1 block. It also removes some test code that modified the small prox response to not reflect the actual response we get.

Next Steps:
- If we see conditionally loading requirejs as acceptable, how do we do it?
  - Where do we host requirejs, using the one hosted by Orbit creates a dependency on Orbit, we should probably host it ourselves
  - We need to use something like react helmet to hoist the script to the head and de-dupe it for stories with more than one IDT1 include

**Edit**
We could use the hosted version of require I found in the IDT2 fallback code: `https://news.files.bbci.co.uk/include/vjassets/js/vendor/require-2.1.20b.min.js`

**Pull request closed without merging**